autosklearn/metalearning/input/aslib_simple.py

```diff
@@ -143,13 +145,17 @@ def _read_configurations(self, filename):
             csv_reader = csv.DictReader(fh)
 
             configurations = dict()
+            hp_names = self.cs.get_hyperparameter_names()
             for line in csv_reader:
                 configuration = dict()
                 algorithm_id = line["idx"]
                 for hp_name, value in line.items():
                     if not value or hp_name == "idx":
                         continue
-
+                    if hp_name not in hp_names:
+                        # skip meta learning configuration
```

Comment:
I believe this comment is not correct, we don't skip the configuration, we only skip the hyperparameter, right?